src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig

```diff
@@ -271,9 +264,7 @@
             <div class="tab-content">
                 {% for child in profile.children %}
                     <h3>
-                        <a href="{{ path('_profiler', { token: child.token }) }}">
-                            {{ helper.set_handler(child.getcollector('request').controller) }}
-                        </a>
+                        {{ helper.set_handler(child.getcollector('request').controller) }}
```

Comment:
While at it; removed a useless outer link, as the actual (inner) link comes from `set_handler`.